### PR TITLE
Various UI fixes

### PR DESCRIPTION
**Pull request by spolu**
- Remove large spacing above titles in apps, data sources, and keys that were inconsistent with other spaces in the app (settings, new app, new data source)
- Harmonize space before list of apps or data-sources or keys
- Fix button to create new key (workding + wrong icon)
- Fix a bit of wording in managed data sources

### CODE REVIEW

**Comment by spolu** [comment]
I thiiiink we might want to also be responsive later down the size of screens for the buttons next to the titles for apps/data sources/keys. Might tackle in follow-up PR.

**Comment by spolu** [comment]
We'll also need to fix the responsiveness of the lateral margins which is inconsistent across the app. Still a lot to iterate on!

File: front/pages/w/[wId]/ds/index.tsx [hunk 1]
```diff
@@ -377,7 +375,7 @@ export default function DataSourcesView({
       </div>
       <div className="mx-auto space-y-4 divide-y divide-gray-200 px-6 sm:max-w-2xl lg:max-w-4xl">
         <div className="sm:flex sm:items-center">
-          <div className="mt-16 sm:flex-auto">
+          <div className="mt-8 sm:flex-auto">
```
Comment (lasryaric) [inline front/pages/w/[wId]/ds/index.tsx:378]: Aligning margin with Apps screen.

**Review by lasryaric — APPROVED** [review]
Voila, A little margin fix. Here is how it looks like now.
Let me know what you think

Screen Shot 2023-05-19 at 21 04 51
Screen Shot 2023-05-19 at 21 05 07
Screen Shot 2023-05-19 at 21 06 00
Screen Shot 2023-05-19 at 21 06 04

**Comment by spolu** [comment]
@lasryaric hmmm there seems to be an alignment problem of the connect button on the managed data sources in your screenshot no? (Which I don’t see on current main)

is that your commit or mine?

File: front/pages/w/[wId]/ds/index.tsx [hunk 2]
```diff
@@ -363,9 +363,7 @@ export default function DataSourcesView({
                             {ds.description}
                           </p>
                         </div>
-                        <div className="mt-2 flex items-center text-sm text-gray-300 sm:mt-0">
-                          <p></p>
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/pages/w/[wId]/ds/index.tsx]: The div had an alignment rôle no?

**Comment by lasryaric** [comment]
>@lasryaric hmmm there seems to be an alignment problem of the connect button on the managed data sources in your screenshot no? (Which I don’t see on current main)

I tried switching between main, your commit and my commit with the terminal on the left and the web browser on the right, the alignment (on the X, horizontal axis)  of the connect button seems to be exactly the same.

>The div had an alignment rôle no?
Tried with and without with term on the left side and screen on the right side, nothing moves at all. This spaces is taken by the parent div anyway.

**Comment by spolu** [comment]
Maybe I’m hallucinating it (on mobile). Will double check once I’m back at the keyboard 👍

**Pull request merged**